_observing-your-data/trace/ta-dashboards.md

```diff
@@ -1,25 +1,143 @@
+
+The **Trace Analytics** application includes two options: **Services** and **Traces**:
+
+  - **Services** lists all services in the application, plus an interactive map that shows how the various services connect to each other. In contrast to the dashboard (which helps identify problems by operation), the **Service map** helps you identify problems by service based on error rates and latency. To access this option, go to **Trace Analytics** > **Services**.
+  - **Traces** groups traces together by HTTP method and path so that you can see the average latency, error rate, and trends associated with a particular operation. For a more focused view, try filtering by trace group name. To acces this option, go to **Trace Analytics** > **Traces**. From the **Trace Groups** panel you can drill down on the traces that make up a trace group. From the **Traces** panel you can drill down on individual traces for a detailed summary.
+
+ **Step 4: Perform correlation analysis**
+  - Select **Services correlation** to reveal connections between various telemetry signals. This allows you to navigate from the logical service level to the associated metrics and logs for that specific service.
+
+---
+
+## Schema dependencies and assumptions
+
+The plugin requires [Data Prepper]({{site.url}}{{site.baseurl}}/data-prepper/) to process and visualize OTel data and relies on the following Data Prepper pipelines for OTel correlations and service map calculations:
+
+- [Trace analytics pipeline]({{site.url}}{{site.baseurl}}/data-prepper/common-use-cases/trace-analytics/)
+- [Service map pipeline]({{site.url}}{{site.baseurl}}/data-prepper/pipelines/configuration/processors/service-map-stateful/)
+
+### Standardized telemetry data
+
+The plugin requires telemetry data to follow the OTel schema conventions, including the structure and naming of spans, traces, and metrics as specified by OTel, and to be implemented using [Simple Schema for Observability (SS4O)]({{site.url}}{{site.baseurl}}/observing-your-data/ss4o/).
+
+### Service names and dependency map
+
+For accurate service mapping and correlation analysis, adhere to the following guidelines:
+
+- Service names must be unique and used consistently across application components.
+- The `serviceName` field is populated using the Data Prepper pipeline.
+- Services must be ingested with predefined upstream and downstream dependencies to construct accurate service maps and understand service relationships.
+
+### Trace and span IDs
+
+Traces and spans must have consistently generated and maintained unique identifiers across distributed systems to enable end-to-end tracing and accurate performance insights.
+
+### RED metrics adherence
+
+The plugin expects metrics data to include rate, error, and duration (RED) indicators for each service, either pre-aggregated using the Data Prepper pipeline or calculated dynamically based on spans. This allows for effective computation and display of key performance indicators.
```

Comment:
Last sentence: "This allows you to effectively compute and display key performance indicators"?